pkg/rke2/psp/psp.go

```diff
@@ -0,0 +1,443 @@
+package psp
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/rancher/spur/cli"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/policy/v1beta1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/transport"
+)
+
+const (
+	defaultRetries          = 5
+	defaultWaitSeconds      = 5
+	defaultTimeout          = 30
+	k8sWrapTransportTimeout = 30
+)
+
+const (
+	kubeSystemNamespace = "kube-system"
```

Comment:
This const isnt necessary, use `metav1.NamespaceSystem`

Reply:
Thanks!